supplementary_style_guide/glossary_terms_conventions/general_conventions/u.adoc

```diff
@@ -47,7 +47,7 @@
 [discrete]
 [[undertow]]
 ==== image:images/yes.png[yes] undertow subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "undertow" subsystem is used to configure the JBoss EAP web server and servlet container settings. Write in lowercase in general text. Use "Undertow subsystem" when referring to the undertow subsystem in titles and headings. See the xref:webhttp-undertow[WebHTTP - Undertow] entry for the correct usage when referring to the undertow subsystem in the management console.
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _undertow subsystem_ is used to configure the JBoss EAP web server and servlet container settings. Write in lowercase in general text. Use "Undertow subsystem" when referring to the undertow subsystem in titles and headings. See the xref:webhttp-undertow[WebHTTP - Undertow] entry for the correct usage when referring to the undertow subsystem in the management console.
```

Comment:
Hi @bergerhoffer . On products that I've worked on to date, I generally surround a subsystem name in backticks when referring to it in a general way, because it's an established component of a product. For example: 

> when referring to the `undertow` subsystem in titles and headings

WDYT?

Reply:
Yep that works! Let's just make sure it gets done in the other letters' PRs too. I'll update throughout here